torchqdynamics/mesolve/adaptive.py

```diff
@@ -15,4 +15,10 @@ def __init__(self, H: TDOperator, jump_ops: Tensor, solver_options: SolverOption
 
     def forward(self, t: float, rho: Tensor) -> Tensor:
         """Compute drho / dt = L(rho) at time t."""
-        return lindbladian(rho, self.H, self.jump_ops)
+        # non-hermitian Hamiltonian
+        H_nh = self.H - 0.5j * self.sum_nojump
+
+        # compute Lindblad(t) @ rho
+        H_nh_rho = H_nh @ rho
+        L_rho_Ldag = kraus_map(rho, self.jump_ops)
+        return -1j * (H_nh_rho - H_nh_rho.adjoint()) + L_rho_Ldag
```

Comment:
Why not using the Lindbladian directly?

Reply:
Because `lindbladian` was recomputing the `sum_nojump` operator every time which is not needed. So it was faster to rewrite it directly inside here.